pandas/tests/test_series.py

```diff
@@ -3065,7 +3065,7 @@ def test_quantile_multi(self):
         assert_series_equal(result, expected)
 
         result = self.ts.quantile([])
-        expected = pd.Series([], name=self.ts.name, index=Index([], dtype=float))
```

Comment:
With `dtype=float`, there is this error
```
AssertionError: Index are different

Index classes are different
[left]:  Index([], dtype='object')
[right]: Float64Index([], dtype='float64')
```
which seems logical (`self.ts.quantile([])` doesn't cast to a `Float64Index`).

This used to work before, when I had last tested ( Nov 11) without the change.